.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,163 @@
+name: Lighthouse Audit
+
+on:
+  deployment_status:
+    states: ['success']
+
+env:
+  VERCEL_PROTECTION_BYPASS: ${{ secrets.VERCEL_PROTECTION_BYPASS_CATALYST_LATEST }}
+  BIGCOMMERCE_ACCESS_TOKEN: ${{ secrets.BIGCOMMERCE_ACCESS_TOKEN }}
+  BIGCOMMERCE_STORE_HASH: ${{ secrets.BIGCOMMERCE_STORE_HASH }}
+
+jobs:
+  generate-lighthouse-audit:
+    name: Lighthouse Audit
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    if: ${{ contains(fromJson('["Production – catalyst-latest", "Preview – catalyst-latest"]'), github.event.deployment_status.environment) }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Pull Request Details
+        id: pr_details
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const prData = require('./.github/workflows/pull-request-data.js');
```

Comment:
We need to add this file back: https://github.com/bigcommerce/catalyst/blob/62d178e61070b4c9be431fe4a02c9344e8b2500c/.github/workflows/pull-request-data.js